spec/mongoid/token/finders_spec.rb

```diff
@@ -56,4 +62,15 @@ class AnotherDocument; include Mongoid::Document; field :token; end
     Mongoid::Token::Finders.define_custom_token_finder_for(AnotherDocument)
     AnotherDocument.find(["1234", "5678"]).should == [document, document2]
   end
+
+  describe :deprecations do
+    it "includes `find`" do
+      class AnotherDocument; include Mongoid::Document; field :token; end
+      document = AnotherDocument.create! :token => "1234"
```

Comment:
Useless assignment to variable - `document`.<br>Use the new Ruby 1.9 hash syntax.